tests/test_form.py

```diff
@@ -212,11 +219,12 @@ def test_form_noaction():
 </html>
 '''
 
+
 def test_form_action():
     browser, url = setup_mock_browser()
-    browser.open_fake_page(submit_form_action, url="http://example.com/invalid/")
+    browser.open_fake_page(submit_form_action, url="http://fake.com/invalid/")
```

Comment:
example.com was really the right thing to do here. fake.com does exist, while example.com is specifically reserved for such usage: https://tools.ietf.org/html/rfc2606 / http://example.com/.

Reply:
Huh! You learn something new every day! :D